components/Footer/index.js

```diff
@@ -1,19 +1,21 @@
-import { StyledFooter, StyledHomeLink, StyledAddChildLink } from "./Footer.styled";
+import { StyledFooter, StyledLink } from "./Footer.styled";
+import { useRouter } from "next/router";
 
 export default function Footer() {
+    const router = useRouter();
     return (
         <StyledFooter>
-            <StyledHomeLink href="/">
-            <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="#016e82" viewBox="0 0 16 16">
+            <StyledLink $active={router.route === "/" ? true : false} href={"/"}>
```

Comment:
In case you prefer a shorter version of the same logic, you can drop the ternary.
```suggestion
            <StyledLink $active={router.route === "/"} href={"/"}>
```